src/Goerli-DssSpellCollateral.sol

```diff
@@ -19,6 +19,44 @@ pragma solidity 0.6.12;
 // pragma experimental ABIEncoderV2;
 
 import "dss-exec-lib/DssExecLib.sol";
+import "dss-interfaces/dss/ChainlogAbstract.sol";
```

Comment:
this won't be required, following the suggestions above (to use directly `DssExecLib` for the chainlog changes).